lib/language_pack/helpers/bundler_wrapper.rb

```diff
@@ -37,8 +37,38 @@ class LanguagePack::Helpers::BundlerWrapper
 
   BLESSED_BUNDLER_VERSIONS = {}
   BLESSED_BUNDLER_VERSIONS["1"] = "1.17.3"
-  BLESSED_BUNDLER_VERSIONS["2"] = "2.3.25"
-  BUNDLED_WITH_REGEX = /^BUNDLED WITH$(\r?\n)   (?<major>\d+)\.\d+\.\d+/m
+  # Heroku-20's oldest Ruby verison is 2.5.x which doesn't work with bundler 2.4
+  BLESSED_BUNDLER_VERSIONS["2.3"] = "2.3.25"
+  BLESSED_BUNDLER_VERSIONS["2.4"] = "2.4.22"
+  BLESSED_BUNDLER_VERSIONS["2.5"] = "2.5.6"
+  BLESSED_BUNDLER_VERSIONS.default_proc = Proc.new do |hash, key|
+    if key.start_with?("1")
+      hash["1"]
+    elsif key.start_with?("2")
```

Comment:
Same comment as above. This would pass for `20.1`, `200.1`, etc.

```suggestion
    elsif key.start_with?("2.")
```
or
```suggestion
    elsif Gem::Version::new(key).segments.first == 2
```